python/ccsds/space_packet_time_stamp_adder.py

```diff
@@ -197,7 +201,7 @@ def handle_msg(self, msg_pmt):
                 self.ascii_dec_num = 1
 
             if self.time_format == 3: # ASCII A
-                arr = self.now.isoformat()
+                arr = self.now.replace(tzinfo=None).isoformat()
```

Comment:
Good call with `.replace(tzinfo=None)` here, since otherwise we would get the timezone formatted into the packet, which would be incorrect.